### PR TITLE
Adding hk in translations

### PR DESCRIPTION
**Pull request by navjotNSK**
## What does this PR do?

It adds the hongkong in all translations file.

Fixes https://github.com/appwrite/appwrite/issues/7880

## Test Plan

No test required

## Related PRs and Issues

* https://github.com/appwrite/appwrite/issues/7880

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

Yes

### CODE REVIEW

File: app/config/locale/translations/af.json [hunk 1]
```diff
@@ -100,6 +100,7 @@
     "countries.gd": "Grenada",
     "countries.gt": "Guatemala",
     "countries.gy": "Guyana",
+    "countries.hk": "Hongkong",
```
Comment (stnguyen90) [inline app/config/locale/translations/af.json:103]: incorrect

File: app/config/locale/translations/bg.json [hunk 2]
```diff
@@ -100,6 +100,7 @@
     "countries.gd": "Гренада",
     "countries.gt": "Гватемала",
     "countries.gy": "Гвиана",
+    "countries.hk": "Хонконг",
```
Comment (stnguyen90) [inline app/config/locale/translations/bg.json:103]: Isn't this missing a space?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
@navjotNSK, thanks for raising this PR! Unfortunately a lot of the translations don't quite seem right and there are more translations than the 11 you've done.

File: app/config/locale/translations/bg.json (hunk 2 above)
Comment (navjotNSK) [reply app/config/locale/translations/bg.json]: space added and all files refactored.

File: app/config/locale/translations/af.json (hunk 1 above)
Comment (navjotNSK) [reply app/config/locale/translations/af.json]: corrected.

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Please fix the merge conflicts. Once done, re-request a review in the PR:

image

**Comment by navjotNSK** [comment]
@stnguyen90 Merge conflicts removed.

File: app/config/locale/translations/bg.json (hunk 2 above)
Comment (stnguyen90) [reply app/config/locale/translations/bg.json]: I don't see the space here.

File: app/config/locale/translations/be.json [hunk 3]
```diff
@@ -100,6 +100,7 @@
     "countries.gd": "Грэнада",
     "countries.gt": "Гватэмала",
     "countries.gy": "Гаяна",
+    "countries.hk": "Гонконг",
```
Comment (stnguyen90) [inline app/config/locale/translations/be.json:103]: how'd you get this? It looks different than what I see online 🧐

File: app/config/locale/translations/bs.json [hunk 4]
```diff
@@ -100,6 +100,7 @@
     "countries.gd": "Grenada",
     "countries.gt": "Guatemala",
     "countries.gy": "Gvajana",
+    "countries.hk": "Hongkong",
```
Comment (stnguyen90) [inline app/config/locale/translations/bs.json]: Is this missing a space?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
some still seem wrong. would you please make sure to go through all of them and verify they're correct?

File: app/config/locale/translations/be.json (hunk 3 above)
Comment (xuelink) [reply app/config/locale/translations/be.json:103]: Ганконг should be.

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**